docs/website/docs/user-guides/advanced/deploy/_accessing_application.mdx

```diff
@@ -0,0 +1,89 @@
+import CodeBlock from '@theme/CodeBlock';
+import Version from '../../../_version.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+You can now access the application by running `odo describe component`:
```

Comment:
Suggestion to clarify this statement (because, IMO, just running `odo describe component` will not access the application ;-)):
```suggestion
You can now determine how to access the application by running `odo describe component`:
```